Sources/ParseSwift/ParseConstants.swift

```diff
@@ -8,14 +8,6 @@
 
 import Foundation
 
-let kParseVersion = "0.0.1"
-
-#if os(iOS)
-let kPFDeviceType = "ios"
```

Comment:
Did these have to be deleted? You could move this inside `ParseConstants`:
```
enum ParseConstants {
  #if os(iOS)
  static let deviceType = "ios"
  #elseif os(macOS)
  ...
}
```

Reply:
I tried something similar but the compiler didn't like it. Since this wasn't needed anywhere else, I just moved it directly to where it's used.

Of course, if you now of a way to add it, you can make the change in the PR for the storage updates

Reply:
I see, with the static way it works